gapis/api/templates/api_types.go.tmpl

```diff
@@ -0,0 +1,266 @@
+{{/*
+ * Copyright (C) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */}}
+
+{{/* ---- Includes ---- */}}
+{{Include "go_common.tmpl"}}
+{{Include "c_common.tmpl"}}
+
+{{$ | Macro "api_types.go" | GoFmt | Write "api_types.go"}}
+
+{{define "api_types.go"}}
+  {{AssertType $ "API"}}
+
+
+  {{template "Go.GeneratedHeader" (Global "OutputDir")}}
+¶
+
+import (
+    "reflect"
+
+    ϟt "github.com/google/gapid/gapis/service/types"
+)
+
+
+  const api_index = uint64({{$.Index}}) << 59
+  const pseudonym_index = uint64(1 << 55)
+  const reference_index = uint64(2 << 55)
+  const pointer_index =   uint64(3 << 55)
```

Comment:
Strange indentation here.
Maybe it would be nice to align all these const assignments?